.gitignore

```diff
@@ -9,3 +9,4 @@ Starwar/UserSettings/
 Starwar/obj/
 Starwar/Assembly-CSharp.csproj
 Starwar/Assembly-CSharp.csproj.user
+Starwar/Assets/Scripts/Player Control/PlayerUI.cs.meta
```

Comment:
maybe a miss click here?

Reply:
yeah I'm using the desktop app, might have accidentally added to gitignore